### PR TITLE
Article enhancements with info about how DNS record normalization and validation is made

### PR DESCRIPTION
**Pull request by ggalmazor**
I've tried to simplify language by getting less into the weeds without removing essential information. I've also focused on providing more approachable information to make it actionable in case customers encounter issues.

# Enhanced TXT record article

image

# Enhance SPF record article

image

# Enhance CAA record article

image

### CODE REVIEW

File: content/articles/txt-record.markdown [hunk 1]
```diff
@@ -1,21 +1,83 @@
...
+
+## Validation
+
+We consider two different validation scenarios:
+
+### When you provide a deserialized TXT record content
+
+We will consider this validation scenario when the TXT record content you provide is not wrapped in double quotes. This is the most common scenario.
+
+In this scenario, there are few limitations on what you can do beyond a hard limit of 1000 characters for the serialized version of your content. See the next section to understand what we mean about the serialized version of the TXT record content.
```
Comment (jacegu) [inline content/articles/txt-record.markdown:38]: Maybe it's worth explaining that in this scenario we will handle the normalization of the the content.

File: content/articles/txt-record.markdown [hunk 2]
```diff
@@ -1,21 +1,83 @@
...
+| [`DKIM` records](/articles/dkim-record) | This record stores important information used in the validation of email in transit.                                                                                       |
+| DMARC records                           | Domain-based Message Authentication Reporting and Conformance records mitigate phishing and spoofing email attacks.                                                        |
+| [`SPF` record](/articles/spf-record/)   | This record is used to indicate to mail exchanges which hosts are authorized to send mail for a domain. TXT records should be used instead of the special SPF record type. |
+| Site Verification Records               | This record proves ownership of a domain and can be used to associate services such as Microsoft 365 and G-Suite to a specific domain.                                     |
+
+## Validation
+
+We consider two different validation scenarios:
+
+### When you provide a deserialized TXT record content
```
[the remark this answers is not in the document]
Comment (ggalmazor) [reply content/articles/txt-record.markdown:34]: Yeah, let's try something different.

File: content/articles/txt-record.markdown (hunk 1 above)
Comment (ggalmazor) [reply content/articles/txt-record.markdown:38]: Good idea!

**Comment by ggalmazor** [comment]
@jacegu I've applied changes based on your feedback. I've kept just the concepts of serialized/deserialized while removing mentions to super technical details like RData or character string. I think this might work and I think keeping the serialized/deserialized semantics could be important as a reference when dealing with customer support requests

**Review by jacegu — APPROVED** [review]
Thanks for the changes. I think this is more clear.

**Comment by ggalmazor** [comment]
@jacegu can I ask you to do a sanity check again, please? I just updated the articles with the latest developments around chunk detection and formatting in TXT records

**Review by jacegu — APPROVED** [review]
Recent changes also look good to me 👍

**Review by nestorsalceda — APPROVED** [review]
Thank you!

**Review by itsalyse — APPROVED** [review]

**Pull request merged**